server/server.go

```diff
@@ -397,52 +360,6 @@ func CreateDB(db *sql.DB) {
 		log.Println("Error: Unable to run migration scripts, execution failed.")
 		log.Fatalln(err)
 	}
-
```

Comment:
No longer inspect the database stacks table, we may at some point revisit stacks but this code is stale and causes issues when developing locally.